### ISSUE TITLE
gr.Examples with image and text don't make a proper table when used with Blocks

### ISSUE THREAD

**Issue by apolinario** [gradio-app/gradio#1948]
### Describe the bug

If using blocks, when I have this code block
```python
input_img = gr.Image()
input_prompt = gr.Textbox()
gr.Examples([["kitty.jpg", "a kitten running around"]],[input_img,input_prompt])
```
I get only the first element, so just the image
![image](https://user-images.githubusercontent.com/788417/182839895-489d9fbb-6298-4848-93c6-b215fc4e76fb.png)

However, if I do a `gr.Examples` using gr.Interfaces it works as it was supposed to: 

![image](https://user-images.githubusercontent.com/788417/182839681-f817faee-6ea2-422e-84fd-2940db4f4167.png)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

With gr.Blocks(), where you can see it failing: 
```python
import gradio as gr
       
with gr.Blocks() as demo:
            with gr.Row():
                with gr.Column():
                    input_img = gr.Image(label="Original", type="file", interactive=False, value="kitty.jpg")
                    input_prompt = gr.Textbox()
                    gr.Examples([["kitty.jpg", "a kitten running around"]],[input_img,input_prompt])
                
demo.launch()
```
![image](https://user-images.githubusercontent.com/788417/182839895-489d9fbb-6298-4848-93c6-b215fc4e76fb.png)

With gr.Interface(), where you can see it working:
```python
import gradio as gr 

def predict():
    pass

image = gr.inputs.Image()
text = gr.inputs.Textbox()

gr.Interface(
    fn=predict,
    inputs=[image, text],
    outputs=gr.outputs.Image(),
    examples=[['kitty.jpg','a kitten running around']]).launch()
```
![image](https://user-images.githubusercontent.com/788417/182839681-f817faee-6ea2-422e-84fd-2940db4f4167.png)

### System Info

```shell
Gradio 3.1.3
```

### Severity

annoying

**Comment by dawoodkhan82** [comment]
@apolinario thanks for reporting. I can confirm this is an issue and can reproduce it. @aliabid94 Ik you worked on examples, can you take a look?

**Comment by abidlabs** [comment]
I can take this, as I'm also looking into #1875

**Comment by abidlabs** [comment]
Hi @apolinario, looks like the issue actually stems from the fact that some of the components in your Blocks demo have labels while others don't. If both the `Image` and `Textbox` components have labels (or neither do), then the issue does not occur.

Nevertheless, opened a PR to fix this: #1952 

Thanks for creating the issue!
